src/twisted/web/resource.py

```diff
@@ -214,8 +214,23 @@ def putChild(self, path, child):
         intended to have the root of a folder, e.g. /foo/, you want
         path to be ''.
 
+        @param path: A single path component.
+        @type path: C{bytes}
```

Comment:
This should be L{bytes}

Reply:
Are you sure? `C{}` and `L{}` are both used for rendering code / identifiers, it's just that `C{}` doesn't attempt to link, right? The other references to non-Twisted identifiers in that file, including `bytes`, use `C{}` (which makes sense to me; there's not much point in linking that identifier to the Python documentation for the `bytes` type).

Reply:
Se: https://twistedmatrix.com/documents/current/core/development/policy/coding-standard.html#docstrings

Reply:
Hm, that's pretty clear (even if it's not how I'd do it myself)! I'll make the change.